pkg/hubble/parser/options/options.go

```diff
@@ -17,6 +17,13 @@ type Options struct {
 	CacheSize         int
 	RedactHTTPQuery   bool
 	RedactKafkaAPIKey bool
+	RedactHttpHeaders *HttpHeadersList
+}
+
+// HttpHeadersList contains the allow/deny list of headers
+type HttpHeadersList struct {
+	Allow []string
```

Comment:
These should be maps.

Reply:
Done in [842a01a](https://github.com/cilium/cilium/pull/26724/commits/842a01aa97d28ef5dc003bb657d2ef13cabcf1d4).